website/src/pages/index.tsx

```diff
@@ -305,7 +306,7 @@ function Configure() {
                   Container Engines
                 </h2>
                 <p className="leading-relaxed text-base">
-                  Handle multiple container engines at the same time (Podman, Docker, Lima...).
+                  Handle multiple container engines at the same time: Podman, Docker, Lima, Kubernetes.
```

Comment:
```suggestion
                  Handle multiple container engines at the same time: Podman, Docker, Lima, and Kubernetes.
```

Reply:
Is that correct?

Reply:
I would say that Kubernetes is not a container engine from Podman Desktop point of view.

it's not plugged at the same level.

Reply:
From an engineer perspective I agree.
But for the user it makes sense: what does it connect to, what can you see in the interface. Do you agree @benoitf?

Reply:
maybe k8s is already covered by 

![image](https://user-images.githubusercontent.com/436777/212092322-b6aaff55-f437-47e2-be56-992f369d9295.png)

Reply:
That's exactly where I find it confusing: why is Kubernetes listed here and not in the container engines list? 

Can you help me clarify? I need to understand the project direction here.

* What the user has: an application to display and interact with pods, containers, images, volumes, and abstract the location of these things. Do we really need to distinguish Kubernetes from the others here?

* What is happening from an engineering perspective: I understand we have 2 sorts of citizens. What / where is the distinction? can you define "not plugged at same level"?
  * Podman, Docker, Lima: controlled via a socket? Currently depicted as "Container engines". Is that name a source of confusion?
  * Kubernetes: controlled via an API?

Reply:
Would "Build" vs. "Run" be the main differentiator?

Reply:
Maybe something like this?

> Workload location
>
> Handle multiple workload location at the same time: Kubernetes instances, and container engines such as Podman, Docker, Lima.

Reply:
And move the container engine list to the *Build, run and manage containers.* section.